src/groovy/org/pih/warehouse/importer/CSVUtils.groovy

```diff
@@ -140,7 +181,7 @@ class CSVUtils {
         // FIXME default value should be localized, but presently is "EA" everywhere
         def numerator = quantityUom ?: "EA"
         def denominator = formatInteger(quantityPerUom ?: 1)
-        return "${quantityUom}/${quantityPerUom}"
+        return "${numerator}/${denominator}"
```

Comment:
Nitpick - this isn't really a case of numerator vs denominator. It's a standard uom notation. for example, "BX/10" is a more succinct version of "1 Box of 10 EA" or "1 Box = 10 EA".

Reply:
Easy to change it to e.g. `return "${formattedUom}/${formattedQuantity}"`

Reply:
resolved in dc92b52b2